src/openrct2/title/TitleSequence.cpp

```diff
@@ -30,24 +30,22 @@
 #include <memory>
 #include <vector>
 
-static std::vector<utf8*> GetSaves(const utf8* path);
-static std::vector<utf8*> GetSaves(IZipArchive* zip);
-static std::vector<TitleCommand> LegacyScriptRead(utf8* script, size_t scriptLength, std::vector<utf8*> saves);
+static std::vector<std::string> GetSaves(const utf8* path);
+static std::vector<std::string> GetSaves(IZipArchive* zip);
+static std::vector<TitleCommand> LegacyScriptRead(utf8* script, size_t scriptLength, const std::vector<std::string>& saves);
 static void LegacyScriptGetLine(OpenRCT2::IStream* stream, char* parts);
 static std::vector<uint8_t> ReadScriptFile(const utf8* path);
 static std::string LegacyScriptWrite(TitleSequence* seq);
 
-TitleSequence* CreateTitleSequence()
+std::unique_ptr<TitleSequence> CreateTitleSequence()
```

Comment:
I question if we need this function anymore.

Reply:
I 'changed' it for an empty constructor.